install/kubernetes/cilium/templates/cilium-operator/service.yaml

```
@@ -1,4 +1,4 @@
-{{- if and .Values.operator.enabled .Values.operator.prometheus.enabled .Values.operator.prometheus.serviceMonitor.enabled }}
+{{- if and .Values.operator.enabled .Values.operator.prometheus.enabled .Values.operator.prometheus.metricsService }}
```

```suggestion
{{- if and .Values.operator.enabled .Values.operator.prometheus.enabled (or .Values.operator.prometheus.serviceMonitor.enabled .Values.operator.prometheus.metricsService) }}
```

This also needs to be gated via OR

---

Thanks. Made changes as suggested.